plugins/lookup/output.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+name: output
+author: Polona Mihalič (@PolonaM)
+version_added: 1.0.0
+short_description: Read state file outputs.
+description:
+    - This lookup returns all outputs or selected output in state file.
+options:
+  _terms:
+    description:
+    - Names of the outputs to return.
+    - If value is set to C(all), all outputs will be returned.
+    required: True
+  project_path:
+    description:
+      - The path to the root of the Terraform directory with the terraform.tfstate file.
+    type: path
```

Comment:
Is it possible to omit `project_path` and `state_file`?  I'd like to have a fallback for this parameter through the `TF_DATA_DIR` environment variable if it is available and `cwd` if not.

Reply:
For the project_path it is possible to omit it, but I would still leave state_file to have a possibility to lookup state file in a custom path (similar like it is done in terraform_output module)

Reply:
Implemented